addons/web/static/tests/views/list_tests.js

```diff
@@ -5225,6 +5225,273 @@ QUnit.module('Views', {
         list.destroy();
     });
 
+    QUnit.test('cell-level keyboard navigation in non-editable list', async function (assert) {
+        assert.expect(15);
+
+        var list = await createView({
+            View: ListView,
+            model: 'foo',
+            data: this.data,
+            arch: '<tree><field name="foo" required="1"/></tree>',
+            intercepts: {
+                switch_view: function (event) {
+                    assert.strictEqual(event.data.res_id, 3,
+                        "'switch_view' event has been triggered");
+                },
+            },
+        });
+
+        assert.ok(document.activeElement.classList.contains('o_searchview_input'), 'default focus should be in search view');
+        // TODO: for some reason, triggering "tab" on the search input doesn't
+        // switch focus to the create button in tests while it works on live
+        $('.o_list_button_add').focus();
+        assert.ok(document.activeElement.classList.contains('o_list_button_add'),
+            'focus should now be in create button');
+        await testUtils.fields.triggerKeydown($(document.activeElement), 'down');
+        // TODO: maybe the header shouldn't be skipped from here ?
```

Comment:
todo

Reply:
The header is not skipped anymore (supposedly), so the next assertion should fail... :thinking: